types/halogenium/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for halogenium 2.3
+// Project: https://github.com/kirillDanshin/halogenium#readme
+// Definitions by: Weslley Nascimento Rocha <https://github.com/WeslleyNasRocha>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
+
+import * as React from 'react';
+
+export type VerticalAlign =
+  | 'baseline'
+  | 'length'
+  | 'sub'
+  | 'super'
+  | 'top'
+  | 'text-top'
+  | 'middle'
+  | 'bottom'
+  | 'text-bottom'
+  | 'initial'
+  | 'inherit';
+
+export interface HalogenCommonProps {
+  loading?: boolean;
+  color?: string;
+  id?: string;
+  className?: string;
+  verticalAlign?: VerticalAlign;
+}
+
+export interface SizeLoaderProps extends HalogenCommonProps {
+  size?: number | string;
+}
+
+export interface MarginLoaderProps extends HalogenCommonProps {
+  margin?: number | string;
+  size?: number | string;
+}
+
+export interface RadiusLoaderProps extends MarginLoaderProps {
+  height?: number | string;
+  width?: number | string;
+  radius?: number | string;
+}
+
+/**
+ * React components
+ */
+export type PulseLoader = React.Component<MarginLoaderProps>;
```

Comment:
Instead of having a separate type alias and const exported for each of these, I'd suggest doing the following:
```ts
export class PulseLoader extends React.Component<MarginLoaderProps> { }
```